extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -698,4 +766,29 @@ export class Bind {
 
     return false;
   }
+
+  /**
```

Comment:
Hmm, maybe dispatch an test-only event (or allow setting of a callback function). You can check `getMode().test` which should excise it from the prod build.